examples/sdbus_async/gatt_props.py

```diff
@@ -0,0 +1,192 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright (C) 2023  Shin'ichiro Kawasaki <kawasaki@juno.dti.ne.jp>
+
+# This example shows how to get properties of GATT services and characteristics
+# of a BLE device.
+# It does:
+# - Discover the specified BLE device with BlueZ D-Bus Adapter API [1]
+# - Connect to the device with BlueZ D-Bus Device API [2]
+# - Get services and characteristics of the device with D-Bus introspection
+# - Get properties of services and characteristics with BlueZ D-Bus GATT API [3]
+# - Also read the characteristics if it has "read" flag.
+
+# [1] https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/doc/adapter-api.txt
+# [2] https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/doc/device-api.txt
+# [3] https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/doc/gatt-api.txt
+
+from __future__ import annotations
+
+import sys
+import xml.etree.ElementTree as ET
+from asyncio import run, sleep
+from typing import Any
+
+from sdbus import SdBus, sd_bus_open_system
+from sdbus.dbus_proxy_async_interfaces import DbusIntrospectableAsync
```

Comment:
By the way the `DbusInterfaceCommonAsync` inherits from `DbusIntrospectableAsync` and does not need to be imported from private internal module.